src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/receipt/containers/receiptContainer.tsx

```diff
@@ -142,6 +142,12 @@ const ReceiptContainer = (props: IReceiptContainerProps) => {
           pdf={pdf || null}
         />
       }
+      {!isLoading() && applicationMetadata.autoDeleteOnProcessEnd &&
+        <AltinnReceiptSimple
+          body={getLanguageFromKey('receipt.body_simple', language)}
```

Comment:
Consider using ‘getTextFromAppOrDefault’ to allow app developer to override texts with app texts

Reply:
Changed to use suggested function.